packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@woocommerce/eslint-plugin",
+	"version": "1.0.0-beta.0",
+	"description": "ESLint plugin for WooCommerce development.",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"woocommerce",
+		"eslint",
+		"plugin"
+	],
+	"homepage": "https://github.com/woocommerce/woocommerce-admin/tree/main/packages/eslint-plugin/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/woocommerce/woocommerce-admin.git",
+		"directory": "packages/eslint-plugin"
+	},
+	"bugs": {
+		"url": "https://github.com/woocommerce/woocommerce-admin/issues"
+	},
+	"files": [
+		"configs",
+		"rules",
+		"index.js"
+	],
+	"main": "index.js",
+	"dependencies": {
+		"@wordpress/eslint-plugin": "^7.1.0",
```

Comment:
@nerrad do you know why this dependency isn't getting added on `npm install`? The [lint step is failing](https://travis-ci.com/github/woocommerce/woocommerce-admin/jobs/364060678).

>ESLint couldn't find the plugin "@wordpress/eslint-plugin".

Reply:
I think the issue here is that with this being a monorepo, and `@woocommerce/eslint-plugin` being included in the top `package.json` via `files://` (so you're always using the latest from the package in development), you have to hoist the dependencies of the new package to the root `package.json` as devDependencies so they are installed.

The important thing is to just make sure the versions match between the package and the root.